src/pages/product_details/ProductDetails.tsx

```diff
@@ -159,7 +159,7 @@ const ProductDetails = (): JSX.Element => {
       setProductReviews((reviews) => reviews.filter((review) => review.reviewId !== reviewId));
       fetchProduct();
     } catch (error) {
-      console.error('An error occurred while deleting review:', error);
+      handleError(error, 'An error occurred while deleting review.');
```

Comment:
Please do not display the internal error to the user

Reply:
yap yap yap